### ISSUE TITLE
Why pixi.js?

### ISSUE THREAD

**Issue by sngo25** [alexweber/es6-jspm-gulp-boilerplate#17]
I see `"pixi.js": "npm:pixi.js@3.0.6",` in the `config.js`. Shouldn't it be here?

**Comment by alexweber** [comment]
@heroandtn3 Good call, that's not supposed to be there!  Wanna remove it? :)  Thanks!
